src/tests/basic/test_arit.cpp

```diff
@@ -597,8 +598,9 @@ void test_pow()
     assert(std::abs(rcp_static_cast<const RealDouble>(r2)->i - 0.501187233627) < 1e-12);
 }
 
- void test_log()
- {
+void test_log()
+{
+    RCP<const Basic> x = symbol("x");
```

Comment:
That's exactly what `symbol("x")` does. It's better to use functions like that, instead of raw `new` directly, that way there is only a few places where raw `new` and `rcp` is used, thus easy to check that they are correct.